pkg/kstatus/polling/engine/engine_test.go

```diff
@@ -16,10 +16,9 @@ import (
 	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
 	"k8s.io/apimachinery/pkg/runtime/schema"
 	"sigs.k8s.io/cli-utils/pkg/kstatus/polling/event"
-	"sigs.k8s.io/cli-utils/pkg/kstatus/polling/testutil"
 	"sigs.k8s.io/cli-utils/pkg/kstatus/status"
 	"sigs.k8s.io/cli-utils/pkg/object"
-	fakemapper "sigs.k8s.io/cli-utils/pkg/testutil"
+	"sigs.k8s.io/cli-utils/pkg/testutil"
```

Comment:
Please make this unrelated change in another PR.

Reply:
This is actually related because I had to move the NewFakeRESTMapper in order to avoid a dependency cycle when using it outside of kstatus. But It could be moved just to make this PR smaller.

Reply:
reverted